integration/tests/cook/util.py

```diff
@@ -228,7 +229,11 @@ def wait_until_inner():
         return wait_until_inner()
     except:
         final_response = query()
-        logger.info(f"Timeout exceeded waiting for condition. Details: {final_response}")
+        try:
+            details = final_response.content
+        except AttributeError:
```

Comment:
Is there a better way to do this? Maybe using `isintance`?

Reply:
I think this is one of those places where [EAFP](https://docs.python.org/3.6/glossary.html#term-eafp) applies.